src/main/java/com/google/gerrit/extensions/client/ListChangesOptionByVersion.java

```diff
@@ -0,0 +1,45 @@
+// Copyright (C) 2012 The Android Open Source Project
```

Comment:
Is this an official Gerrit class? Only such classes should be in `com.google.gerrit` (it's almost a plain copy of the Gerrit extensions tree).

Reply:
Makes sense, moved it.